README.md

```diff
@@ -11,23 +11,17 @@
 - Yarn
 - [egghead-systems](https://github.com/eggheadio/egghead-systems)
 
-## Endpoints
-
-Endpoints are run locally via the `egghead-systems` project. To run them:
+## Environments
 
 - Copy `.env_template` to `.env`.
-- Copy the JWT from the query param at `http://egghead.dev:5000/users/jwt?return_to=http://localhost:3000`.
-- Add this JWT to `.env`.
-
-### Environments
-
-You can swap out the value of `REACT_APP_EGGHEAD_BASE_URL` in `.env` for different environments. You will need a JWT for each environment. For example, you could copy the JWT from the query param at `https://egghead.io/users/jwt?return_to=https://instructor.egghead.io` for production.
+- For each environment in `.env`, paste the JWT you get from `{environment}/users/jwt?return_to=http://localhost:3000`.
 
 ## Scripts
 
 _These assume you have `egghead-systems` running locally_
 
 - `yarn` to install latest packages.
+- `yarn api-fake` to start the fake environment API server.
 - `yarn start` to compile and lint the app; open `localhost:3000` to view.
 - `yarn guide` to compile the living style guide; open `localhost:9009` to view.
 - `yarn test` to run tests.
```

Comment:
Next steps: Make `yarn start` do everything you need to work:

I would like `yarn start` to also run `yarn guide` with something like [npm-run-all](https://www.npmjs.com/package/npm-run-all).

I would like to add a `yarn api-dev` script to run the vagrant steps needed to get `egghead-systems` running (assuming this repo is cloned as a sibling to `egghead-rails` and `egghead-systems`).

I would like `yarn start [environment]` to additionally automatically run `yarn api-fake` or `yarn api-dev` depending on `environment`.